.travis.yml

```diff
@@ -42,13 +47,14 @@ script:
 - make test
 - make crossbuild
 - make smoke
-- if [[ $TRAVIS_PULL_REQUEST = 'false' && $DOCKER_LOGIN_PASSWORD && $DOCKER_LOGIN_USERNAME ]]; then
-    make docker-build smoke-docker docker-push;
-  fi
-- if [[ $TRAVIS_PULL_REQUEST = 'false' && $PACKAGECLOUD_TOKEN ]]; then
-    make package smoke-package;
-  fi
+
+after_success:
+  - echo $TRAVIS_PULL_REQUEST
+  - echo $TRAVIS_BRANCH
+  - sed -i s/@@BRANCH/$TRAVIS_BRANCH/ bin/send-docker-hub-trigger.sh
+  - bin/send-docker-hub-trigger.sh
```

Comment:
I think this script should instead be `./script/trigger-docker-hub-build` and should accept the branch as a positional argument instead of altering it with `sed`.

Reply:
Moved from `bin/` to `script/` and removed `.sh` extension. (See below re: `sed`.)